ArjunaJTA/maven/README.md

```diff
@@ -1,44 +1,39 @@
-JBoss, Home of Professional Open Source Copyright 2008, Red Hat Middleware 
```

Comment:
removed copyright?

Reply:
the copyright was left as it was just it's displayed as the comment